### PR TITLE
feat: populate section on TOC integration in PACT data model

### PR DESCRIPTION
**Pull request by raimundo-henriques**

### CODE REVIEW

File: specs/index.bs [hunk 1]
```diff
@@ -1829,7 +1829,97 @@ Note: Chapter [[#appendix-a-sf-example]] contains an example.
 
 ### TOC ### {#pcf-mapping-toc}
 
-Issue: this sections is still work in progress
+Note: This chapter refers to the PACT Data Model. See [[!PACTDX]] Chapter 4 for further details.
+
+A TOC CAN be integrated into the Pathfinder Data Model ([[!PACTDX]]) by storing a <{TOC}> as an
+extension (see [[!DATA-MODEL-EXTENSIONS]]) in a `ProductFootprint`.
+
+For interoperability reasons, additional attributes of such a `ProductFootprint` SHOULD be derived
```
Comment (zeitgeist) [inline specs/index.bs:1837]:
```suggestion
For interoperability reasons, several attributes of a `ProductFootprint` MUST be derived
```

File: specs/index.bs [hunk 2]
```diff
@@ -1829,7 +1829,97 @@ Note: Chapter [[#appendix-a-sf-example]] contains an example.
...
+from the <{TOC}>. This is specified in the table below.
```
Comment (zeitgeist) [inline specs/index.bs:1838]: we should link to the table here

File: specs/index.bs [hunk 3]
```diff
@@ -1829,7 +1829,97 @@ Note: Chapter [[#appendix-a-sf-example]] contains an example.
...
+        <td>`unitaryProductAmount`
+        <td>
+              MUST be set to `"1"` so that the ProductFootprint represents the emissions per ton kilometer of the TOC.
+
+      <tr>
+        <td>CarbonFootprint
+        <td>`pCfExcludingBiogenic`
+        <td>
+              MUST be set to the logistics emissions intensity of the TOC, excluding biogenic emissions,
+              found in <{TOC/co2eIntensityWTW }>.
```
Comment (zeitgeist) [inline specs/index.bs:1888]:
```suggestion
              defined in <{TOC/co2eIntensityWTW }>.
```

**Review by zeitgeist — APPROVED** [review]
Looks good to me modulo the inline comments

File: specs/index.bs (hunk 2 above)
Comment (raimundo-henriques) [reply specs/index.bs:1838]: Agreed! I'll take the opportunity and do the same for ShipmentFootprint

**Review by zeitgeist — APPROVED** [review]
LGTM. Thank you, @raimundo-henriques

**Pull request merged**